careers/application-process/index.html

```diff
@@ -281,13 +281,17 @@ <h3 class="u-hide-on-mobile">Asking About your Application</h3>
     <div class="block
                 block__padded-top
                 block__border-top
+                block__flush-bottom
                 content-l">
-        <section class="content-l_col content-l_col-1-2">
+        <article class="content-l_col content-l_col-1-3">
+            {% include "_template-current-openings.html" %}
+        </article>
+        <article class="content-l_col content-l_col-1-3">
             {% include "_template-working-at-cfpb.html" %}
-        </section>
-        <section class="content-l_col content-l_col-1-2">
+        </article>
```

Comment:
I could see how "independently distributable" could be interpreted to mean modules that would be replicated in a website, but the whole description is `The article element represents a self-contained composition in a document, page, application, or site and that is intended to be independently distributable or reusable, e.g. in syndication.` That seems to describe things would be captured by an RSS feed like blog posts, or the main "content" body of a page.